### ISSUE TITLE
Unable to run in CentOS/Tomcat9 - Can anyone identify an OS/Tomcat that it works in?

### ISSUE THREAD

**Issue by bobjase** [siteadmin/reference-ccda-validator#46]
We are trying to run the newest WAR file and we get an OutOfMemory exception (same as jxfish).

Can someone tell us a platform they have this working on?

**Comment by drbgfc** [comment]
One of the latest successful combinations used by SITE is:
Ubuntu 1 4.04.5 LTS
apache-tomcat-8.0.43

**Comment by bobjase** [comment]
Dan,

Thanks for the update. What about my other questions from the previous post:

1. What version of Java are you using?
2. Where do we get the VSAC files that you are using?

**Comment by bobjase** [comment]
Here is a 7zip compressed version of 20150301 (not quite 20150901 but it was the best I could find).

I don't know yet if it will work and we're still waiting on Dan to know what version of Java to try, but in case someone else is struggling with this, maybe it will help a little.

[sct2_Description_Full-en_US1000124_20150301.zip](https://github.com/siteadmin/reference-ccda-validator/files/2676658/sct2_Description_Full-en_US1000124_20150301.zip)

**Comment by drbgfc** [comment]
1: 
Here is what one of our servers is using that works:
openjdk version "1.8.0_171"

2:
Most of the code systems and valuesets are externally published and this project does NOT come packaged with any of these externally published files. However, valuesets which were not externally published by VSAC were hand created by SITE and are available for download here: https://github.com/siteadmin/code-validator-api/tree/master/codevalidator-api/docs/ValueSetsHandCreatedbySITE

The external ones need to be obtained via the VSAC website.

**Comment by drbgfc** [comment]
The zip provided appear to be corrupt. What exactly is it supposed to contain?

**Comment by bobjase** [comment]
Github doesn't allow uploading of .7z files so I renamed it to .zip but, as I wrote above, it was compressed with 7zip (so you will need 7zip to decompress). 

It contains the SNOMED FULL codes from 20150301. Not quite 20150901 but close and its 194MB decompressed (which is what you said your codebase is). If you are willing to share your 20150901 here, that would be even better (I'm not sure why they aren't in the github repository)

**Comment by drbgfc** [comment]
Understood on the compression extension, sorry I missed that.

The single hardest issue for users is getting the external vocabulary. I understand that it is a struggle. But, at this time, due to licensing we cannot provide them. They must be obtained from the source (typically the creator of the codeSystem or valueSet). We don't have the auuthority to do so.

I feel like this is getting off-base from the bug title (which is resolved correct?). Once you have obtained the file we are using, please create a new ticket if you are having an OOME or other error. I apologize that we don't have very much time right now to devote to local deployment issues. I will try find the time though.

**Comment by bobjase** [comment]
Dan,

- Can you provide a list of the ValueSets that need downloading?
- It is not clear to be where, on the VSAC site, to download anything. Can you point us in the right direction?

**Comment by drbgfc** [comment]
Did you read this readme and download and view all of the files at this link?
https://github.com/siteadmin/code-validator-api/tree/master/codevalidator-api/docs
